fontforge/featurefile.c

```diff
@@ -5696,11 +5672,7 @@ static void fea_ParseGDEFTable(struct parseState *tok) {
 	    }
 	    for (;;) {
 		fea_ParseTok(tok);
-		if ( tok->type==tk_int )
-		    /* Not strictly cricket, but I'll accept it */;
-		else if ( tok->type==tk_char && tok->tokbuf[0]=='<' )
-		    fea_ParseCaret(tok);
-		else
```

Comment:
Dunno, I felt it would be good to be able to drop the deprecated syntax. Maybe I can bring it back if needed.